core/storage/statistics/gae_models.py

```diff
@@ -741,8 +901,17 @@ def get_master_model(cls, exploration_id, exploration_version, state_name):
     @classmethod
     def get_all_models(cls, exploration_id, exploration_version, state_name):
         """Retrieves all models and shards associated with the specific
-        exploration state. Returns None if no answers have yet been submitted to
-        the specified exploration state.
+        exploration state.
+
+        Args:
+            exploration_id: str. The exploration ID.
+            exploration_version: int. The version of the exploration to fetch
+                answers for.
+            state_name: str. The name of the state to fetch answers for.
+
+        Returns:
+            list(entities)|None. Returns None if no answers have yet
```

Comment:
Shouldn't this be `list(StateAnswersModel)|None`?

Reply:
ohh sorry. I changed this.